config/locales/event.en.yml

```diff
@@ -45,3 +46,8 @@ en:
     mark_as_vetting_in_progress: Mark as vetting in progress
     mark_as_ready_to_copy: Mark as ready to copy
     mark_as_ready_to_dispatch: Mark as ready to dispatch
+    mark_as_waiting_for_data__desc: Case is waiting for data
+    mark_as_ready_for_vetting__desc: Case is ready for vetting
+    mark_as_vetting_in_progress__desc: Vetting in progress
+    mark_as_ready_to_copy__desc: Case is ready to copy
+    mark_as_ready_to_dispatch__desc: Case is ready to dispatch
```

Comment:
Great :) - this hasn't come up before, but just so you know we try to keep keys in alphabetical order in files, as much as possible. Would you mind moving these to appropriate locations?

Reply:
You'll usually spot other things out of place when you start looking - up to you whether you fix any "broken windows" on the way past. It's really good to do it, cleaning things as you go even if they're not yours.... 

 - on the other hand, it's better to keep a PR focused on just what the ticket is about. 

So your call! Ideally do any other improvements when you see them but if there's more than just one or two, I'd open a separate PR we can just fast-track

Reply:
@Polsonby,  the changes I made is not for fixing any "broken windows", it is outcome of trying to add flexible extra-description for an event without breaking existing logical, ideally I would like to extend the properties of an event, but the way I had in my mind will change the structure of event.yml too much,  I can explain it tomorrow if you would like a call

Reply:
Sorry, that "broken windows" remark was not about your PR so far - it was  about when if you start looking, you'll see lots of places where things _aren't_ in alphabetical order and the hypothetical situation you might find yourself in where you might want to add other cases you saw in passing when you're changing the order of yours. Does that make sense? Let's get on a call tomorrow - sorry for not being clear enough!